static/.htaccess

```diff
@@ -30,3 +30,13 @@ FileETag MTime
 <FilesMatch "\.txt$">
     FileETag None
 </FilesMatch>
+
+# CORS only for mozilla domains
```

Comment:
This comment led me to believe it was allowing _any_ Mozilla domain to access our static resources, but I think it's actually allowing only SUMO itself if it's on a Mozilla domain.